app/src/main/java/unipd/se18/ocrcamera/BlurObject.java

```diff
@@ -0,0 +1,47 @@
+package unipd.se18.ocrcamera;
+
+import android.graphics.Bitmap;
+
+
+/** Class that holds the bitmap, the name , and the blur value calculated
+ *  Pratesi Leonardo
+ *
+ */
+
+public class BlurObject {
+    String name;
+    Bitmap image;
+    double blur;
+
+    public BlurObject(String name, Bitmap image, double blur)
+    {   this.name=name;
+        this.image= image;
+        this.blur=blur;
+    }
+
+    public BlurObject(String name, Bitmap image)
+    {
+        this.name=name;
+        this.image=image;
+        this.blur= CameraActivity.blurValue(this.image);

```

Comment:
You should document what this method does as I have no access to the documentation

Reply:
Yes thanks